repositories.yaml

```diff
@@ -0,0 +1,10 @@
+mobile_metrics_example:
+  app_name: mobile_metrics_example
+  os: Android
+  notification_emails:
+    - frank@mozilla.com
+  url: 'https://github.com/fbertsch/mobile-metrics-example'
+  histogram_file_paths:
+    - Histograms.json
```

Comment:
Why not parse the Histograms.yaml?
Histograms.json has the old format, with no categories. Can we just not support that?

Reply:
I thought we had discussed supporting both, but extend this later to include `Histograms.yaml`. There's a dependency on the `Histograms` class in `python_moztelemetry`. Let me see how much work that might be.